static/js/template/controllers/template.controller.js

```diff
@@ -47,62 +47,77 @@
     self.templateComponents = [
       {
         id: 1,
-        name: "Label",
+        name: "Project",
         icon: null,
         type: 'label',
-        description: "Use for static text: labels, headings, paragraphs"
+        description: "Title of the Project. For example, if you were designing a website, this would be "Design a Website for me.""
       },
+      { 
+	id: 2, 
+	name: "Objective", 
+	icon: null, 
+	type: 'label', 
+	description: "One task associated with the project. For example, if you were designing a website, an example objective could be "Create a navigation bar in the header."" 
+      },
+      {
+	id: 3, 
+	name: "Task",
```

Comment:
this will confuse people, we will figure this out in a different way, if it's a macro project we will skip Task Design and just give them a text area